src/improvements/tasks/sep/000-opcm-upgrade-v200/VALIDATION.md

```diff
@@ -333,7 +333,7 @@ For each contract listed in the state diff, please verify that no contracts or s
 
     Raw Slot:          0x0000000000000000000000000000000000000000000000000000000000000005
     Raw Old Value:     0x0000000000000000000000000000000000000000000000000000000000000000
-    Raw New Value:     0x00000000000000000000000000000000000000000000000000000000008baa6f
+    Raw New Value:     0x00000000000000000000000000000000000000000000000000000000008f6971
```

Comment:
This and the previous value are not expected to be constant, as they are updated any time a dispute game is resolved.

You can see how these values [are set](https://specs.optimism.io/fault-proof/stage-one/fault-dispute-game.html#anchor-state) in the new ASR by the OPCM.

And here you can see how the [current ASR is updated](https://github.com/ethereum-optimism/optimism/blob/8d0dd96e494b2ba154587877351e87788336a4ec/packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol#L1019) by a dispute game. 

We should add a note to indicate that.

Reply:
Yes, it says below: 

> The 'Raw New Value' for this entry might be different than what is seen in the Tenderly state diff.

Reply:
Right, I just did a lot of copy and paste to find the diff, but didn't read the descriptions for each one.